POMs/loginPage.ts

```diff
@@ -9,69 +9,60 @@ export class LoginPage {
     readonly loginErrorMessage: Locator;
     readonly forgotPasswordLink: Locator;
     readonly logoutLink: Locator;
-    readonly logoutModal: Locator; 
+    readonly logoutModal: Locator;
     readonly homePageButton: Locator;
-    
-    
 
-        
-            constructor(page: Page) {
-                this.page = page;
-                this.emailInputField = page.getByRole('textbox', { name: 'Email Address' });
-                this.passwordInputField = page.getByRole('textbox', { name: 'Password' });
-                this.loginButton = page.getByRole('button', { name: 'Log in' });
-                this.loginErrorMessage = page.getByText('Wrong email or password.');
-                this.forgotPasswordLink = page.getByRole('link', { name: 'Forgot Password?' });
-                this.logoutLink = page.getByRole('link', { name: 'Log out' });
-                this.logoutModal = page.getByText('Do you want to Log out from the account? Cancel Confirm');
-                this.homePageButton = page.getByRole('link', { name: 'Gymshark', exact: true });
-            }
-        
-    async login(emailValue: string, passwordValue: string){
+    constructor(page: Page) {
+        this.page = page;
+        this.emailInputField = page.getByRole('textbox', { name: 'Email Address' });
+        this.passwordInputField = page.getByRole('textbox', { name: 'Password' });
+        this.loginButton = page.getByRole('button', { name: 'Log in' });
+        this.loginErrorMessage = page.getByText('Wrong email or password.');
+        this.forgotPasswordLink = page.getByRole('link', { name: 'Forgot Password?' });
+        this.logoutLink = page.getByRole('link', { name: 'Log out' });
+        this.logoutModal = page.getByText('Do you want to Log out from the account? Cancel Confirm');
```

Comment:
it would be better if you use unique locators to have more stability (if text get changed)